target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -82,6 +82,9 @@ cf-wr800n)
 	ucidef_set_led_netdev "lan" "lan" "$boardname:white:ethernet" eth0.1
 	set_wifi_led "$boardname:white:wifi"
 	;;
+xzwifi,creativebox-v1)
+	ucidef_set_led_netdev "internet" "internet" "$boardname:blue:internet" "eth0.2"
```

Comment:
First of all, please keep the alphabetical order. Some issue in the other files.

Please use ucidef_set_led_switch instead. As soon as someone decides to use a different vlan for wan, the trigger will not work any longer.